internal/api/v1/routes/routes.go

```diff
@@ -116,6 +122,14 @@ func RegisterRoutes(
 	jobs.Post("/", jobHandler.CreateJob).Name(CreateJob)
 	jobs.Put("/:id", jobHandler.UpdateJob).Name(UpdateJob)
 	jobs.Delete("/:id", jobHandler.TerminateJob).Name(TerminateJob)
+
+	// ---------------------------
+	// User endpoints
+	users := v1.Group("/users")
+	users.Get("/:id", userHandler.GetUserByID).Name(GetUserByID)
+	users.Get("/", userHandler.GetUserByUsername).Name(GetUserByUsername)
```

Comment:
Let's add an option for getting all users and deleting a user. Also per the comment at the top of the file let's reorder these. 

>  Within this ordering, param urls (ie /:id) should go last, otherwise fiber will interpret the route slug as that param.

I think we can make the standard Get method the GetAll method and allow the username body param to be optional. So if it is not passed in, then all users are returned. Then we don't need to add an additional route. 

```suggestion
	users.Get("/", userHandler.GetUsers).Name(GetUsers)
	users.Get("/:id", userHandler.GetUserByID).Name(GetUserByID)
```